cert-data.yml

```diff
@@ -64,6 +82,9 @@
 Default/OpenEdX/Certificate: &certificate_defaults
   VERSION: 4_programmatic
   TEMPLATEFILE: False
+  font: 
+    name: "SourceSansPro"
+    type: "Regular" 
```

Comment:
ack, `SourceSansPro` is not a default font in this repo, we should use: `OpenSans` & `Light`